plugins/channelrx/demodrtty/rttydemodsink.cpp

```diff
@@ -29,7 +29,6 @@
 #include "rttydemodsink.h"
 
 RttyDemodSink::RttyDemodSink(RttyDemod *packetDemod) :
-        m_rttyDemod(packetDemod),
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok